samples/subsys/usb/uvc/prj.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_LOG=y
+CONFIG_LOG_MODE_MINIMAL=y
+CONFIG_VIDEO=y
+CONFIG_VIDEO_BUFFER_POOL_NUM_MAX=4
+CONFIG_VIDEO_BUFFER_POOL_SZ_MAX=7300
+CONFIG_VIDEO_EMUL_IMAGER=y
+CONFIG_VIDEO_EMUL_RX=y
+CONFIG_VIDEO_LOG_LEVEL_DBG=y
+CONFIG_USB_DEVICE_STACK=n
+CONFIG_USB_DEVICE_STACK_NEXT=y
+CONFIG_USBD_VIDEO_CLASS=y
+CONFIG_USBD_VIDEO_LOG_LEVEL_DBG=y
+CONFIG_SAMPLE_USBD_PID=0x000B
```

Comment:
`0x000B` is for the bluetooth HCI sample. For the new USBD_UVC sample, it seems we need to add an entry in `doc/connectivity/usb/device/usb_device.rst` ?

Reply:
I missed it, thank you for checking!